network/endpoint_windows.go

```diff
@@ -242,6 +242,15 @@ func (nw *network) configureHcnEndpoint(epInfo *EndpointInfo) (*hcn.HostComputeE
 		return nil, err
 	}
 
+	if epInfo.NICType == cns.NodeNetworkInterfaceAccelnetFrontendNIC {
```

Comment:
@paulyufan2 What does this NICType signify?

Reply:
It signifies "accelnet NIC type"; we receive NIC type from CNS as well

Reply:
will use another PR for accelnet